### PR TITLE
Changes to webhook descriptions

### PR DESCRIPTION
**Pull request by tatiana-perry**
This is to go over the descriptions to webhooks for better end-user clarity.

### CODE REVIEW

**Comment by tatiana-perry** [comment]
@bookernath & @karen-white Tag anyone else that needs to review this and answer questions.

File: webhooks.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Below are my proposed changes
+
+### Orders
+- store/order/created: Fires if an order is created using the control panel, an app or via the API.
+- store/order/updated: Fires when an already created order is updated. Any changes to an existing order will fire this webhook. 
+Updates can include changing the status, updating a coupon or changing an address. 
+- store/order/statusUpdated: This will only fire if the order status has changed. Such as Pending to Awaiting Payment.
+
+### Products
+- store/product/inventory/updated: Fires if a products inventory is updated. This includes inventory changes due to an order being placed,
```
Comment (karen-white) [inline webhooks.md]: Suggested edit (for typos and clarity):

Fires if a product's inventory is decremented or incremented, including when an order is placed. Webhook responds to inventory updates made using the Control Panel, CSV import, API or an app.

**Comment by tatiana-perry** [comment]
@bc-ppuente @bigcommerce-labs/product-support-engineering 

I am thinking of removing the webhook deleted from the docs. See above ⬆️. Any thoughts or objection to this?

**Comment by bc-ppuente** [comment]
@tatiana-perry - I'm actually not able to register a hook with the "store/order/deleted" scope. I get a 400 response with an error messaging stating "Invalid scope"
screen shot 2018-11-12 at 1 15 40 pm

While I'm able to register a hook with the "store/order/archived" scope, I can't actually trigger it either. I've tried these three actions, none of them trigger a message (though I had a "store/order/*" hook running in parallel and that one did trigger on status updates and order creation):
- Archive an order from the control panel
- Send a DELETE request to an order object (which archives it)
- "Delete" an order by starting a new checkout after attempting payment (i.e. the action that results in a "The incomplete order #N was deleted automatically (customer began new checkout)." log entry)

**Comment by tatiana-perry** [comment]
Archived always takes forever for me. It's pretty inconsistent. So if you want to open a ticket you can. I already opened on for deleted. 
https://github.com/bigcommerce-labs/dev-docs-beta/pull/4#issuecomment-436683760

**Comment by tatiana-perry** [comment]
Proposed changed in staging under review

**Comment by tatiana-perry** [comment]
[DEVDOCS-470](https://jira.bigcommerce.com/browse/DEVDOCS-470)

**Pull request closed without merging**